Stack traces including full path of code on machine used to package release

For example:

```
Installing  Strongback Java Library 1.2.1

   downloading strongback-1.2.1.tar.gz to /Users/rhauch/strongback-archives/strongback-1.2.1.tar.gzopen /Users/rhauch/strongback-archives/strongback-1.2.1.tar.gz: no such file or directory
panic: open /Users/rhauch/strongback-archives/strongback-1.2.1.tar.gz: no such file or directory

goroutine 1 [running]:
panic(0x24a040, 0xc4203cb4d0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
main.(*Environment).InstallRelease(0xc420077c70, 0x7fff5fbff815, 0x5, 0x0, 0x0)
	/User/jane/dev/go/src/strongback.org/cli/main.go:404 +0xcb2
main.main()
	/User/jane/dev/go/src/strongback.org/cli/main.go:815 +0xdfd
```

This can be changed per [this mechanism](https://github.com/golang/go/issues/13809#issuecomment-168874760) by using the `trim path` Go compiler option:

    go build -gcflags=-trimpath=$GOPATH -asmflags=-trimpath=$GOPATH

so that the output is:

```
Installing  Strongback Java Library 1.2.1

   downloading strongback-1.2.1.tar.gz to /Users/rhauch/strongback-archives/strongback-1.2.1.tar.gzopen /Users/rhauch/strongback-archives/strongback-1.2.1.tar.gz: no such file or directory
panic: open /Users/rhauch/strongback-archives/strongback-1.2.1.tar.gz: no such file or directory

goroutine 1 [running]:
panic(0x24a040, 0xc4203cb4d0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
main.(*Environment).InstallRelease(0xc420077c70, 0x7fff5fbff815, 0x5, 0x0, 0x0)
	src/strongback.org/cli/main.go:404 +0xcb2
main.main()
	src/strongback.org/cli/main.go:815 +0xdfd
```

--- a/Makefile
+++ b/Makefile
@@ -13,7 +13,8 @@ BUILD_TIMESTAMP=`date +%FT%T%z`
 BUILD_DATE=`date +"%Y-%m-%d"`
 
 # Setup the -ldflags option for go build here
-LDFLAGS=-ldflags "-w -s -X main.Version=${VERSION} -X main.Date=${BUILD_DATE} -X main.ExecName=${BINARY} -X main.Build=${BUILD_TIMESTAMP}"
+LDFLAGS=-ldflags "-w -s -X main.Version=${VERSION} -X main.Date=${BUILD_DATE} -X main.ExecName=${BINARY} -X main.Build=${BUILD_TIMESTAMP}" 
+GCFLAGS=-gcflags=-trimpath=${GOPATH} -asmflags=-trimpath=${GOPATH}
 
 #
 # Cleans and builds everything
@@ -43,15 +44,15 @@ windows: out/windows package-windows
 #
 out/macos: dependencies
 	$(info macos:   building executable)
-	@GOOS="darwin" GOARCH="amd64" go build ${LDFLAGS} -o ${OUTPUT_DIR}/macos/strongback ${WHAT}
+	@GOOS="darwin" GOARCH="amd64" go build ${LDFLAGS} ${GCFLAGS} -o ${OUTPUT_DIR}/macos/strongback ${WHAT}
 
 out/linux: dependencies
 	$(info Linux:   building executable)
-	@GOOS="linux" GOARCH="amd64" go build ${LDFLAGS} -o ${OUTPUT_DIR}/linux/strongback ${WHAT}
+	@GOOS="linux" GOARCH="amd64" go build ${LDFLAGS} ${GCFLAGS} -o ${OUTPUT_DIR}/linux/strongback ${WHAT}
 
 out/windows: dependencies
 	$(info Windows: building executable)
-	@GOOS="windows" GOARCH="386" go build ${LDFLAGS} -o ${OUTPUT_DIR}/windows/strongback.exe ${WHAT}
+	@GOOS="windows" GOARCH="386" go build ${LDFLAGS} ${GCFLAGS} -o ${OUTPUT_DIR}/windows/strongback.exe ${WHAT}
 
 .PHONY: test
 test:

--- a/Makefile
+++ b/Makefile
@@ -17,15 +17,18 @@ LDFLAGS=-ldflags "-w -s -X main.Version=${VERSION} -X main.Date=${BUILD_DATE} -X
 GCFLAGS=-gcflags=-trimpath=${GOPATH} -asmflags=-trimpath=${GOPATH}
 
 #
+# Make everything
+#
+.PHONY: all
+all: macos linux windows
+
+#
 # Cleans and builds everything
 #
 .PHONY: clean
 clean:
 	$(info Cleaning 'out' directory)
 	@rm -rf ${OUTPUT_DIR}
-
-.PHONY: all
-all: macos linux windows
 
 #
 # Packaging for each OS

